persistence/jpa/src/main/java/org/infinispan/persistence/jpa/configuration/JpaStoreConfigurationBuilder.java

```diff
@@ -29,8 +30,9 @@ public JpaStoreConfigurationBuilder entityClass(Class<?> entityClass) {
       return self();
    }
 
+   @Deprecated
    public JpaStoreConfigurationBuilder batchSize(long batchSize) {
-      attributes.attribute(BATCH_SIZE).set(batchSize);
+      attributes.attribute(MAX_BATCH_SIZE).set((int) batchSize);
```

Comment:
The casting isn't ideal, but at the same time will anybody need a batch > Integer.MAX_VALUE?